coderedcms/utils.py

```diff
@@ -5,11 +5,13 @@
 
 from coderedcms.settings import cr_settings
 
+
 def get_protected_media_link(request, path, render_link=False):
     if render_link:
-        return mark_safe("<a href='{0}{1}'>{0}{1}</a>".format(request.build_absolute_uri('/')[:-1], path))
+        return mark_safe("<a href='{0}{1}'>{0}{1}</a>".format(request.build_absolute_uri('/')[:-1], path))  # noqa
```

Comment:
This line could be easily broken up at the parentheses i.e.
```python
mark_safe(
    "<a href='{0}{1}'>{0}{1}</a>".format(request.build_absolute_uri('/')[:-1], path)
)
```
You could even break up the `.format()` if needed too.